{
	"title": "CVE-2008-4099",
	"cve": "CVE-2008-4099",
	"notes": {
		"description": "PyDNS (aka python-dns) before 2.3.1-4 in Debian GNU/Linux does not use random source ports or transaction IDs for DNS requests, which makes it easier for remote attackers to spoof DNS responses, a different vulnerability than CVE-2008-1447.",
		"published": "2008-09-18",
		"modified": "2023-02-12"
	},
	"references": [
		{
			"url": "http://www.openwall.com/lists/oss-security/2008/09/11/1",
			"description": "MISC:[oss-security] 20080911 Re: CVE Request (ruby -- DNS spoofing vulnerability in resolv.rb)"
		},
		{
			"url": "http://www.openwall.com/lists/oss-security/2008/09/16/4",
			"description": "MISC:[oss-security] 20080915 Re: CVE Request (ruby -- DNS spoofing vulnerability in resolv.rb)"
		},
		{
			"url": "http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=490217",
			"description": "MISC:http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=490217"
		},
		{
			"url": "http://packages.debian.org/changelogs/pool/main/p/python-dns/python-dns_2.3.3-1/changelog",
			"description": "MISC:http://packages.debian.org/changelogs/pool/main/p/python-dns/python-dns_2.3.3-1/changelog"
		}
	]
}
